### DISCUSSION TITLE
Rate limit for using API calls

### DISCUSSION THREAD

**Discussion by Abhishek-A-Jadhav** [simple-login/app#2257] [Q&A]
I'm using the API as a part of my person project. And basically, I've run into the following error:
`Request error: 429 Client Error: TOO MANY REQUESTS for url: https://app.simplelogin.io/api/v3/alias/custom/new`

Now, the error is obvious given the rate at which I am calling the API.

Just want to understand what is the limit to the API calls in a day? Or any specific frequency at which I can make a call?

**Comment by wonderwareguru** [comment]
Pretty underhanded right? You pay for an advertised unlimited aliasing service, no asterisk or disclosures for limits, and whammmm you get hit with one. What kind of user base was simplelogin expecting? The local Parent Teacher Association? Obviously people's main intent with a service like this is bulk signups.
